schema/private.openapi.yaml

```diff
@@ -140,6 +140,34 @@ paths:
         '400':
           $ref: '#/components/responses/BadRequest'
 
+  /internal/v2/send_job_request:
```

Comment:
I'd like to stay as close to the original method names (dispatch) as possible, but I'm not sure how to accomplish that.  Maybe we could add an additional directory (/internal/v2/simplified/dispatch) or call it dispatch_by_host_id (seems weird).  Naming is hard.  :(  Mabye its the v3 version of the endpoint??